<launch>
    <arg name="dev" default="/dev/ttyACM0"/>
    <arg name="baud" default="115200"/>

    <arg name="twist_topic" default="joy_twist"/>
    <arg name="empty_topic" default="joy_empty"/>

    <arg name="camera_topic" default="recv_image_camera"/>
    <arg name="hector_topic" default="recv_image_hector"/>

    <arg name="enable" default="True"/>
    <arg name="enable_reply_ticks" default="False"/>

    <arg name="control_info_topic" default="control_info"/>
    <arg name="pir_string_topic" default="pir_string"/>

    <!-- Locally defined topic -->
    <arg name="reply_topic" value="send_reply"/>

    <node pkg="nordic_node" type="nordic_send.py" name="nordic_send" output="screen">
        <param name="~dev" value="$(arg dev)"/>
        <param name="~baud" value="$(arg baud)"/>
        <param name="~twist_topic" value="$(arg twist_topic)"/>
        <param name="~empty_topic" value="$(arg empty_topic)"/>
        <param name="~reply_topic" value="$(arg reply_topic)"/>
        <param name = "~control_info_topic" value = "$(arg control_info_topic)"/>
        <param name="~enable" value="$(arg enable)"/>
    </node>

    <node pkg="nordic_node" type="nordic_recv.py" name="nordic_recv" output="screen">
        <param name="~dev" value="$(arg dev)"/>
        <param name="~baud" value="$(arg baud)"/>
        <param name="~camera_topic" value="$(arg camera_topic)"/>
        <param name="~hector_topic" value="$(arg hector_topic)"/>
        <param name="~reply_topic" value="$(arg reply_topic)"/>
        <param name="~pir_string_topic" value = "$(arg pir_string_topic)"/>
        <param name="~enable" value="$(arg enable)"/>
        <param name="enable_reply_ticks" value="$(arg enable_reply_ticks)"/>
    </node>

</launch>